examples/config-dynamodb.json

```diff
@@ -0,0 +1,29 @@
+{
+    "distSpecVersion": "1.0.1-dev",
+    "storage": {
```

Comment:
trivy's CVE db is the only one that should take up local disk. If that is disabled in config, can we make sure that nothing is present on local disk except config?

Reply:
Do you mean that nothing should be written on the disk by zot if all the config is cloud-based, and a mechanism to check for that should be implemented? I'm not sure I understand